src/server.js

```diff
@@ -3,13 +3,14 @@ const path = require('path');
 const logRoutes = require('./middleware/log-routes');
 const routes = require('./routes');
 
-const server = express();
+const app = express();
```

Comment:
This feels silly, but "app" really is the standard name for this, and if fellows follow a tutorial they'll have to keep switching back and forth.